### PR TITLE
Feature/rm mangrove coinable

### PR DESCRIPTION
**Pull request by ronnyhaase**
- Remove Magrove campaign
- Add support for partners without website
- Remove Coinable website

### CODE REVIEW

File: packages/app/package.json [hunk 1]
```diff
@@ -89,6 +89,7 @@
     ]
   },
   "devDependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```
Comment (dankelleher) [inline packages/app/package.json:92]: Is this being used?

File: packages/app/src/hub/HubApp.tsx [hunk 2]
```diff
@@ -37,12 +30,11 @@ const _HubApp: ForwardRefRenderFunction<
   const [zenMode, updateZenMode] = useZenMode();
   const { myTree } = useForest();
   const { totalCarbon } = useCarbon();
-  const { needsUpgrade: showLockAlert } = useLockHubDetails();
 
   const [showIntro, updateShowIntro] = useState(false);
   const [introLeft, updateIntroLeft] = useState(false);
   const [showHubNav, updateShowHubNav] = useState(false);
-  const [showAlerts, updateShowAlerts] = useState(false);
+  const [, updateShowAlerts] = useState(false);
```
Comment (dankelleher) [inline packages/app/src/hub/HubApp.tsx:37]: I think this can probably be removed now

File: packages/app/src/hub/HubApp.tsx [hunk 3]
```diff
@@ -257,25 +249,6 @@ const _HubApp: ForwardRefRenderFunction<
               </div>
             </Link>
           </div>
-          <div
-            className={clx(
-              "transition-opacity ease-in duration-500",
-              showHubNav ? "opacity-100" : "opacity-0"
-            )}
-          >
-            <Link to="/lock" className="block w-full mt-4 leading-none">
```
Comment (dankelleher) [inline packages/app/src/hub/HubApp.tsx:266]: Did you mean to remove the lock button?

File: packages/app/package.json (hunk 1 above)
Comment (ronnyhaase) [reply packages/app/package.json:92]: It's hard to catch, but in dev mode there's a warning that the dependency should be explicit.

File: packages/app/src/hub/HubApp.tsx (hunk 3 above)
Comment (ronnyhaase) [reply packages/app/src/hub/HubApp.tsx:266]: Oops, no :)

**Review by dankelleher — APPROVED** [review]

**Pull request merged**